macros/activity_schema/activity/generate_activity_yml.sql

```diff
@@ -0,0 +1,74 @@
+-- Get data_types object from model config block
+{% macro get_activity_config(model) -%}
+    {% set relation = ref(model) %}
+    {% for node in graph.nodes.values()
+        | selectattr("resource_type", "equalto", "model")
+        | selectattr("name", "equalto", relation.identifier) %}
+        {% do return(node.config) %}
+    {% endfor %}
+{%- endmacro %}
+
+{% macro generate_activity_yml(activities) %}
+
+{% set yaml_output = [] %}
+
+{% do yaml_output.append('version: 2') %}
+{% do yaml_output.append('models:') %}
+
+-- Loop through each activity
+{% for activity in activities %}
+
+  {% do yaml_output.append('  - name: ' ~ activity) %}
```

Comment:
Not a requirement, but in case you find it annoying to manually manage whitespace - this package has a `whitespace` macro that you can leverage to specify the amount of leading whitespace you want in each line you insert

Reply:
Thanks for the hint! I had a look at the macro, but I wasn't sure on how to use it correctly or if it would add clarity as there are string whitespaces neccessary as well. I can try to integrate it if you want!

Reply:
Ah this was actually a faulty recommendation on my part - I have other projects that include a macro that allows users to dynamically generate whitespace for tab indentation purposes. The macro in this project is used for parsing aql. I confused the two in my recommendation. Please disregard - apologies for the misdirect!